pkg/cmd/config/config.go

```diff
@@ -18,9 +18,12 @@ func NewCmdConfig(f *cmdutil.Factory) *cobra.Command {
 	longDoc.WriteString("Display or change configuration settings for gh.\n\n")
 	longDoc.WriteString("Current respected settings:\n")
 	for _, co := range config.ConfigOptions() {
-		longDoc.WriteString(fmt.Sprintf("- `%s`: %s", co.Key, co.Description))
+		longDoc.WriteString(fmt.Sprintf("- %s: %s", co.Key, co.Description))
```

Comment:
So this undoes a change from a recent PR by @muzimuzhi as all keywords were being formatted with backticks.  If you look at the [web page version](https://cli.github.com/manual/gh_config), you can see this causes them to be code highlighted:

![Screenshot 2024-01-22 at 1 44 18 PM](https://github.com/cli/cli/assets/2089743/2721e43e-eb7d-4448-ba4a-a5702f2597ec)

```suggestion
		longDoc.WriteString(fmt.Sprintf("- `%s`: %s", co.Key, co.Description))
```